docs/docsite/rst/shared_snippets/galaxy_server_list.txt

```diff
@@ -1,11 +1,19 @@
 
 
+If you use the ``ansible-galaxy`` command, you need to first configure the server lists that command communicates with. You can configure this as follows:
 
-By default running ``ansible-galaxy`` will use the :ref:`galaxy_server` config value or the ``--server`` command line
-argument when it performs an action against a Galaxy server. The ``ansible-galaxy collection install`` supports
-installing collections from multiple servers as defined in the :ref:`ansible_configuration_settings_locations` file
-using the :ref:`galaxy_server_list` configuration option. To define multiple Galaxy servers you have to create the
-following entries like so:
+* Set the server list in the :ref:`galaxy_server_list` configuration option in :ref:`ansible_configuration_settings_locations` file.
+* Use the ``--server`` command line argument to limit to an individual server.
+
+To configure a Galaxy server list:
+
+#. Add the ``server_list``  option under the ``[galaxy]`` section to one or more server names.
```

Comment:
Does `#.` mean a bullet list as well?

Reply:
It's rst fancy for a numbered list :-)  The output looks like this:
![image](https://user-images.githubusercontent.com/30267855/68797154-0d4c9980-0622-11ea-8f72-988071c23a81.png)

Reply:
Ahh cool, TIL